### PR TITLE
Test lowest versions

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

**Comment by WouterJ** [comment]
Anyone knows what's causing these "*PHPCR\RepositoryException: You need to be logged in for this operation*" errors? /cc @lsmith77

**Comment by lsmith77** [comment]
no clue .. I will try to investigate tonight

File: composer.json [hunk 1]
```diff
@@ -12,17 +12,17 @@
...
+        "symfony-cmf/routing-auto": "~1.1",
+        "symfony-cmf/routing-bundle": "~1.2,>=1.2.0",
         "symfony-cmf/core-bundle": "~1.2",
         "aferrandini/urlizer": "1.0.*",
         "symfony/config": "~2.2",
         "jms/metadata": "1.5.*"
     },
     "require-dev": {
-        "symfony-cmf/testing": "~1.1",
+        "symfony-cmf/testing": "dev-fix_container_creation as 1.2.7",
```
Comment (lsmith77) [inline composer.json]: this needs to be reverted to ``"symfony-cmf/testing": "~1.2,>=1.2.7",`` once https://github.com/symfony-cmf/Testing/pull/109 is merged and tagged as 1.2.7

**Comment by lsmith77** [comment]
closing in favor of #158

**Pull request closed without merging**